frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/load_mesh_menu_item.tsx

```diff
@@ -0,0 +1,34 @@
+import { WarningOutlined } from "@ant-design/icons";
+import FastTooltip from "components/fast_tooltip";
+import type { VolumeTracing } from "oxalis/store";
+import type { APIMeshFile } from "types/api_flow_types";
+
+type Props = {
+  currentMeshFile: APIMeshFile | null | undefined;
+  volumeTracing: VolumeTracing | null | undefined;
+};
+
+export function LoadMeshMenuItemLabel({ currentMeshFile, volumeTracing }: Props) {
+  const showWarning =
+    volumeTracing?.volumeBucketDataHasChanged ??
+    (!volumeTracing?.hasEditableMapping && volumeTracing?.mappingIsLocked);
```

Comment:
As written in another comment `volumeTracing?.volumeBucketDataHasChanged` is never set to true by the frontend.

This warning also appears when the tracing has no editable mapping (`volumeTracing?.hasEditableMapping === undefined`) but the user did some annotation `volumeTracing?.mappingIsLocked === true`
As `!undefined` is evaluated to `true` the resulting logic is always true for some basic volume annotation which was edited but does not have an editable mapping. Therefore, the logic is currently working from a user perspective, but the code is not doing what it is intended to do (at least thats how I read it.)

But imo it should be fixed by using the correct field name in the `setVolumeBucketDataHasChangedAction`

Or to avoid such unintended conversions, maybe use `volumeTracing?.hasEditableMapping === false`. I think this would have shown the error right away